frontend/common/access/RolesWizard/steps/RoleAssignmentsReviewStep.tsx

```diff
@@ -0,0 +1,214 @@
+import { useMemo, useState } from 'react';
+import { usePageWizard } from '../../../../../framework/PageWizard/PageWizardProvider';
+import { Badge, Divider, ExpandableSection, Title } from '@patternfly/react-core';
+import { getItemKey } from '../../../crud/Data';
+import {
+  ITableColumn,
+  PageDetail,
+  PageTable,
+  TextCell,
+  useInMemoryView,
+} from '../../../../../framework';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+type ReviewData = {
+  resourceType?: string;
+  resources?: { id: number; name: string; username?: never }[];
+  edaRoles?: { id: number; name: string; description?: string; username?: never }[];
+  teams?: { id: number; name: string; username?: never }[];
+  users?: { id: number; name?: never; username: string }[];
+};
+
+interface ReviewExpandableListProps<
+  K extends { name: string; username?: never } | { name?: never; username: string },
+> {
+  selectedItems: K[];
+  label?: string;
+  fieldName: string;
+}
+
+const StyledDivider = styled(Divider)`
+  margin-bottom: var(--pf-v5-global--spacer--lg);
+`;
+
+const StyledBadge = styled(Badge)`
+  margin-left: var(--pf-v5-global--spacer--sm);
+`;
+
+const StyledTitle = styled(Title)`
+  margin-bottom: 1rem;
+`;
+
+export function RoleAssignmentsReviewStep() {
+  const { wizardData } = usePageWizard();
+  const { t } = useTranslation();
+
+  const reviewData = useMemo<ReviewData>(() => {
```

Comment:
this useMemo might not be necessary

Reply:
Thanks! Replaced with destructuring assignment from `wizardData`.